docs/README.md

```diff
@@ -10,10 +10,10 @@ KubeVirt's primary CRD is the VirtualMachine (VM) resource, which contains a col
 
 ### Project Components
 
- * virt-api: This component provides a HTTP RESTful entrypoint to manage the virtual machines within the cluster.
- * virt-controller: This component manages the state of each VMI within the Kubernetes cluster.
- * virt-handler: This is a daemon that runs on each Kubernetes node. It is responsible for monitoring the state of VMIs according to Kubernetes and ensuring the corresponding libvirt domain is booted or halted accordingly.
- * virt-launcher: This component is a place-holder, one per running VMI. It contains the running VMI and remains running as long as the VMI is defined.
+* **virt-api**: This component provides an HTTP RESTful entry-point to manage the virtual machines within the cluster. It is created by adding the KubeVirt's CRDs.
```

Comment:
I'm not sure if the added sentence is useful. It may confuse the reader - what is created by CRDs, the entry point? or virt-api?

Reply:
Right, I will omit that